### PR TITLE
alphabetize the output of the help command; added short array syntax to the README

### PR DESCRIPTION
**Pull request by elstamey**
- [x] alphabetize the list of commands that is output by help #2 
- [x] clarify the formatting of the configuration in the README
- [x] verify the test runs correctly

### CODE REVIEW

**Comment by elstamey** [comment]
I was able to get this code to work in the most backwards way.  I updated my code, pushed it to the branch, ran composer in another space, ran phergie with my config file there, and executed the help command.  I had to do it this way because:
- my scaffold environment(from the hackathon) isn't letting me run the bot with the config
- my plugin in the scaffold environment won't let me run the tests because it says Phake isn't installed.

I need some advice again on this dev environment because each time I want to fix something, I spend a lot of time trying to figure out how to run the plugin, test it, etc.

File: composer.json [hunk 1]
```diff
@@ -15,8 +15,10 @@
         "phergie/phergie-irc-plugin-react-command": "~1"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.1.*",
-        "phake/phake": "2.0.0-beta2"
+        "phpunit/phpunit": "4.5.*",
+        "phake/phake": "@stable",
+        "phergie/phergie-irc-plugin-react-bofh": "^1",
+        "elstamey/phergie-irc-plugin-react-tableflip": "dev-master"
```
Comment (elazar) [inline composer.json]: Think these last two plugins need to be taken out.

File: tests/PluginTest.php [hunk 2]
```diff
@@ -11,10 +11,13 @@
 namespace Phergie\Irc\Tests\Plugin\React\CommandHelp;
 
 use Phake;
+use ReflectionMethod;
```
Comment (elazar) [inline tests/PluginTest.php]: Why is this needed?

File: tests/PluginTest.php [hunk 3]
```diff
@@ -160,6 +163,31 @@ public function testGetSubscribedEvents()
...
+    {
+        // We have to set the private method accessible
+        $method = new \ReflectionMethod(
+            '\Phergie\Irc\Plugin\React\CommandHelp\Plugin', 'alphabetize'
+        );
+        $method->setAccessible(TRUE);
+
+        //Test our method with a few plugins
+        $this->assertEquals(
+            'bofh tableflip',
```
Comment (elazar) [inline tests/PluginTest.php]: Instead of including two external plugins, it should be possible to implement some sort of mocking to return specific command names for this test case.

File: composer.json (hunk 1 above)
Comment (elstamey) [inline composer.json]: yes.  I was manually testing since Phake wouldn't run at all.

File: tests/PluginTest.php (hunk 3 above)
Comment (elstamey) [inline tests/PluginTest.php]: Still working on that bit.  I was just able to run tests for the package and, I believe mock a plugin. I'm slowly getting there and will remove the bits I needed to test manually.  Not being able to even run the test suite was an inconvenience.
